packages/ethereum-contracts/ops-scripts/deploy-framework.js

```diff
@@ -578,14 +591,19 @@ module.exports = eval(`(${S.toString()})({skipArgv: true})`)(async function (
         },
         async () => {
             let superTokenFactoryLogic;
-            const helper = await web3tx(
-                SuperTokenFactoryHelperLogic.new,
-                "SuperTokenFactoryHelperLogic.new"
-            )();
+            const superTokenLogic = useMocks
+                ? await web3tx(SuperTokenLogic.new, "SuperTokenLogic.new")(
+                      superfluid.address,
+                      0
+                  )
+                : await web3tx(
+                      SuperTokenLogic.new,
+                      "SuperTokenLogic.new"
+                  )(superfluid.address);
```

Comment:
I guess the way this is structured, the script would always deploy a new instance of SuperTokenLogic, even if only the factory code changed.
Wouldn't break anything, but potentially waste gas.
I suggest we leave it like this for now and unbundle when modifying further for NFT deployment.

Reply:
let's do the fine tuning in ERC20x branch.